### ISSUE TITLE
Record and show activity logs

### ISSUE THREAD

**Issue by inglesp** [opensafely-core/airlock#140]
It is important that we have a paper trail of all changes to requests and files.  This is important for transparency (X requested these files to be released on and Y and Z approved) and for debuggability.

For each of the following events:

* creation of release request
* addition/withdrawal of file to release request
* approval/rejection of file by output checker (note that an output checker may change their mind)
* release of files

we should record who did the event, when, and the path to the file.

And we should show a simple table of events, either on the request page, or linked to from there.

Depends on #137 and #138.

**Comment by inglesp** [comment]
Closing in favour of #169.
